qiita_pet/handlers/api_proxy/prep_template.py

```diff
@@ -193,13 +194,14 @@ def prep_template_post_req(study_id, user_id, prep_template, data_type,
 
     msg = ''
     status = 'success'
+    prep = False
```

Comment:
Is this supposed to be `prep = None`? otherwise prep is getting different types (bool and PrepTemplate)